src/modules/secret-sharing/secret-sharing.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Body, Controller, Get, Param, ParseIntPipe, Post } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { SecretSharingService } from './secret-sharing.service';
+import { CreateSecretSharingDto } from './dtos/create-secretSharing.dto';
+
+
+@ApiTags('secret-sharing')
+@Controller('secret-sharing')
+export class SecretSharingController {
+    constructor(private readonly secretsSharingService: SecretSharingService) {}
+  
+    @Get(':accountId')
```

Comment:
please with a const

Reply:
it will be updated on the next pull request